src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -129,4 +129,223 @@
   <data name="InvalidOperation_EnumOpCantHappen" xml:space="preserve">
     <value>Enumeration has either not started or has already finished.</value>
   </data>
+  <data name="Serialization_NonSerType" xml:space="preserve">
+    <value>Type '{0}' in Assembly '{1}' is not marked as serializable.</value>
+  </data>
+  <data name="Argument_DataLengthDifferent" xml:space="preserve">
+    <value>Parameters 'members' and 'data' must have the same length.</value>
+  </data>
+  <data name="ArgumentNull_NullMember" xml:space="preserve">
+    <value>Parameters 'members' and 'data' must have the same length.</value>
```

Comment:
Or, just delete this resource string altogether and update the current uses...

Instead of:

```c#
throw new ArgumentNullException(nameof(members), SR.Format(SR.ArgumentNull_NullMember, i));
```

Something like:

```c#
throw new ArgumentNullException(nameof(members) + "[" + i + "]");
```